exercises/D-strings/exercise.js

```diff
@@ -1,3 +1,7 @@
 // Start by creating a variable `message`
 
+let message = [] ;
```

Comment:
I know this test passed, but I would normally expect message to be a string and not an array.